website/src/libs/api-v3/firebase.js

```diff
@@ -0,0 +1,68 @@
+import Firebase from 'firebase';
+import nconf from 'nconf';
+const IS_PROD = nconf.get('IS_PROD');
+const FIREBASE_CONFIG = nconf.get('FIREBASE');
+const FIREBASE_ENABLED = IS_PROD && FIREBASE_CONFIG.ENABLED === 'true';
+
+let firebaseRef;
```

Comment:
I think this should just be 'firebase'. That's the normal convention for declaring instances of an object.